src/Compilers/CSharp/Portable/Symbols/Source/SourceParameterSymbolBase.cs

```diff
@@ -93,7 +93,7 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
 
             if (type.ContainsNullableReferenceTypes())
             {
-                AddSynthesizedAttribute(ref attributes, compilation.SynthesizeNullableAttribute(type));
+                AddSynthesizedAttribute(ref attributes, moduleBuilder.SynthesizeNullableAttribute(this, type));
```

Comment:
>AddSynthesizedAttribute [](start = 16, length = 23)

Should SynthesizeIsReadOnlyAttribute call be moved to this method?

Reply:
Moved.